cache/compression/compression.go

```diff
@@ -212,13 +211,9 @@ func (a *Archiver) compressWithBinary(archivePath string, includePaths []string)
 
 	a.logger.Debugf("$ %s", cmd.PrintableCommandArgs())
 
-	out, err := cmd.RunAndReturnTrimmedCombinedOutput()
+	_, err := cmd.RunAndReturnTrimmedCombinedOutput()
```

Comment:
If we don't care about the output, why don't we use another method such as `Run()`? 🤔

Reply:
Yes and no (confusing footgun) as at least the error collector would return the last error even if the output is not logged.